lib/TPP/Dialect/Check/CheckDialect.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The IREE Authors
```

Comment:
We cannot attribute copyright to other authors. This line has to go.

Keep them all like the LLVM ones, since we want these files to be in LLVM soon,